pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -71,27 +71,20 @@ final ArgParser _parser = (() {
   // UI to avoid a painful migration.
   parser.addMultiOption('platform',
       abbr: 'p',
-      help: 'The platform(s) on which to run the tests',
-      defaultsTo: [
-        'vm'
-      ],
-      allowed: [
-        'vm',
-        for (var runtime in allRuntimes) runtime.identifier
-      ],
-      allowedHelp: {
-        'vm': Runtime.vm.description,
-        for (var runtime in allRuntimes)
-          runtime.identifier: runtime.description,
-      });
+      help: 'The platform(s) on which to run the tests.\n'
```

Comment:
I had to revert my change to use `allowed` and `allowedHelp`, even though the output is much nicer, we don't know about the custom runtimes at this point.